tools/wptrunner/wptrunner/update/sync.py

```diff
@@ -10,37 +10,6 @@
 
 here = os.path.abspath(os.path.split(__file__)[0])
 
-bsd_license = """W3C 3-clause BSD License
```

Comment:
@jgraham is this part of Gecko sync? Will there be any adverse impact of not continuing to write this file?

Reply:
I don't think so. This dates from when there was a choice of licenses and we wanted to indicate that we were taking the BSD option. In practice gecko doesn't use this code anyway, but I think servo does, so @jdm may want to be aware that it's changing.